### PR TITLE
Autolathe fix, pipe things, recycler check, some qol & peeking

### PR DESCRIPTION
**Pull request by PrPleGoo**
Fixes issue #233.
Nerfs autolathe crafting speed.
Adds reinforced pipes. Additional sanity check on the recycler. 
Makes injector and reinforced pipes available in the pipe dispenser. 
Meter now reads the data back to you if you slap it point blank. 
TEG circulators can no longer input from a source whose pressure is equal to the output. 
Regular pipes are now worse at resisting pressure, so bad you can break them with the high powered pump. 
Adds peek verbs, fixes peek not resetting on movement. 
Adds coder sprites for reinforced pipes.

### CODE REVIEW

**Comment by PrPleGoo** [comment]
Hold off on accepting this PR until I fixed everything in #233

**Comment by Zero-Bits** [comment]
I've restarted the Travis build for you.

**Comment by Zero-Bits** [comment]
Still looks good and passes Travis. I don't personally agree with 2-way canisters but the code checks out, so it still has my approval.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -862,7 +862,7 @@
 	icon_state = "roburger"
 	filling_color = "#cccccc"
 	center_of_mass = "x=16;y=11"
-	nutriment_desc = list("bun" = 2, "metal" = 3)
+	nutriment_desc = list("bun" = 2, DEFAULT_WALL_MATERIAL = 3)
```
Comment (Zero-Bits) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: I would not describe the taste of a roburger as DEFAULT_WALL_MATERIAL.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -1148,7 +1148,7 @@
 	trash = /obj/item/stack/rods
 	filling_color = "#fffee0"
 	center_of_mass = "x=17;y=15"
-	nutriment_desc = list("tofu" = 3, "metal" = 1)
+	nutriment_desc = list("tofu" = 3, DEFAULT_WALL_MATERIAL = 1)
```
Comment (Zero-Bits) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Seriously. Not in the food taste descriptions.

**Pull request merged**